tensorflow/lite/micro/tools/make/targets/ceva_makefile.inc

```diff
@@ -0,0 +1,102 @@
+
+
+TARGET_ARCH :=
+
+ifeq ($(TARGET_ARCH), )
+  $(error TARGET_ARCH must be specified on the command line)
+
+endif# Create a cflag based on the specified TARGET_ARCH. For example:
+#   TARGET_ARCH=CEVA_BX1 --> -DCEVA_BX1
+#   TARGET_ARCH=CEVA_SP500 --> -DCEVA_SP500
+TARGET_ARCH_DEFINES := -D$(shell echo $(TARGET_ARCH) | tr [a-z] [A-Z])
+
+ TARGET_TOOLCHAIN_PREFIX := ceva
+ CXX_TOOL = clang++
+ CC_TOOL = clang
+ LD_TOOL = ceva-elf-ld
+ LD = ceva-elf-ld
+  
+  PLATFORM_ARGS += \
+  $(TARGET_ARCH_DEFINES) \
+    -c \
+	-fmessage-length=0 \
+	-fpermissive \
+	-Os \
+	-g3 \
+	-Wall \
+	-pedantic \
+	-D_LIBCPP_INLINE_VISIBILITY="" \
+	-D_LIBCPP_EXTERN_TEMPLATE_INLINE_VISIBILITY=""
+	 
+  
+
+ifeq ($(TARGET_ARCH), CEVA_BX1)
+ #TARGET_ARCH := 
+CXXFLAGS := -std=c++11 -DTF_LITE_STATIC_MEMORY 
+CCFLAGS  := -std=c11   -DTF_LITE_STATIC_MEMORY 
+
+
+  PLATFORM_ARGS += \
+     --target=cevabx1-elf \
+	-mcpu=cevabx1v1.0.0 \
+	-m32x32 \
+	-mgetbits \
+	-mloop-buffer-size=10 \
+	-mfp=1 \
+	-mdpfp=1
+
+
+  CXXFLAGS += $(PLATFORM_ARGS)
+  CCFLAGS += $(PLATFORM_ARGS)
```

Comment:
this can be common to all the architechtures, move to line 91?

Reply:
Yes. Done